apps/ocsp.c

```diff
@@ -111,7 +111,9 @@ typedef enum OPTION_choice {
     OPT_RKEY, OPT_ROTHER, OPT_RMD, OPT_RSIGOPT, OPT_HEADER,
     OPT_V_ENUM,
     OPT_MD,
+# ifdef OCSP_DAEMON
```

Comment:
Why is this change required?

Reply:
With `OPT_MULTI` defined, but not used, builds with strict warnings fail.  This option is not supported when the platform is not capable of fork(), waitpid(), ...

Reply:
Rich reported a build failure:
```
    o ../openssl/apps/pkcs8.c
   ../openssl/apps/ocsp.c:251:17: error: enumeration value 'OPT_MULTI' not handled in switch [-Werror,-Wswitch]
           switch (o) {
                   ^
```

Reply:
Right. A solution more consistent with what we have done elsewhere is to move the guards around the OPT_MULTI condition in the switch to look like this:

````
         case OPT_MULTI:
 # ifdef OCSP_DAEMON
             multi = atoi(opt_arg());
 # endif
             break;
````
This avoids the need for additional guards here. E.g. compare with OPT_TRACE in s_server

Reply:
I wanted a `usage()` error when `-multi` is used but not supported. And leaving the option out of the switch seemed a good way to do that.  Silently ignoring it did not seem right...

Reply:
You don't need to do that.  If you put the ifdef just in the flag/help table, then the case can never happen and a usage message will occur.  Do what Matt said and try it :)

Reply:
Right, I forget the flag table already had the `#ifdef` so, the error will happen anyway.  Pushed updated commit.